handbook/engineering/environments.md

```diff
@@ -0,0 +1,25 @@
+# Environment
+
+This page describes our different cloud environments.
+
+## Google Cloud
+
+We utilize multiple Google Cloud projects and folders to organize our workloads and manage access control.
+
+
+### Root Projects
+
+- **Sourcegraph**: Services for sourcegraph.com.
+- **sourcegraph-code-intel**: Services for Code Intel code execution.
+- **sourcegraph-managed-**: There are multiple `-managed` projects, one for each our managed deployments.
+- **TelligentSourcegraph**: Data pipeline and storage for pings and Cloud event logging.
+- **Sourcegraph Auxiliary**: Testing clusters and deployments.
+- **Universities**: Sourcegraph instances for universities. (Deprecated)
+- **sourcegraph-interviews**: Shared project for interviews.
```

Comment:
Fyi, I created this for the distribution team kubernetes coding exercise (so we can create a cluster for the candidate) but it hasn't really been used. lmk if you think we should keep this.

Reply:
I think we can keep it for now, Ill check what is running in it and the test to ensure its still useful, but in general I like providing a test environment for interviews.